packages/d/devextreme.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "devextreme",
```

```suggestion
  "name": "devextreme",
  "filename": "js/dx.all.js",
```
Let's suggest a default file, my guess would be this?